packages/mobile/src/screens/app-screen/AppTabNavigationProvider.tsx

```diff
@@ -0,0 +1,40 @@
+import type { ReactNode } from 'react'
+import { useMemo, createContext, useState } from 'react'
+
+import type { Nullable } from '@audius/common'
+import type { NativeStackNavigationProp } from '@react-navigation/native-stack'
+
+import type { AppTabScreenParamList } from './AppTabScreen'
+
+type AppTabNavigation = NativeStackNavigationProp<AppTabScreenParamList>
+
+type AppTabNavigationContextValue = {
+  navigation: Nullable<AppTabNavigation>
+  setNavigation: (navigation: AppTabNavigation) => void
+}
+
+export const AppTabNavigationContext =
+  createContext<AppTabNavigationContextValue>({
+    navigation: null,
+    setNavigation: () => {}
+  })
+
+type AppTabNavigationProviderProps = { children: ReactNode }
+
+export const AppTabNavigationProvider = (
```

Comment:
Kinda nitty, but I'm wondering if we should use this opportunity to call it something else besides Tabs which is overloaded with the top tabs swipe thing.

Maybe just AppNavigatorProvider? or BottomBarNavigatorProvider?

Reply:
yeah im down for an update, we have AppTabs still, so that's why this is named the way it is. I'll add a tech-debt item to address all the references to AppTabs? and do samethnig like BottomTabNavigator/AppNavigator or something?

Reply:
sounds good!